### ISSUE TITLE
[bug] Vue: "MetaMask" does not support programmatic chain switching

### ISSUE THREAD

**Issue by creazy231** [reown-com/appkit#3391]
### Link to minimal reproducible example

Just try to call the "switchNetwork" function when connected with MetaMask

### Summary

Calling the `` function like so:
```js
const networkData = useAppKitNetwork();

function handleMinting() {
  if (networkData.value.chainId !== config.networks.default_network.chainId) {
    networkData.value.switchNetwork(sepolia);
  }
}
```
results in `"MetaMask" does not support programmatic chain switching` when connected with MetaMask.

### List of related npm package versions

```
    "@reown/appkit": "^1.5.3",
    "@reown/appkit-adapter-wagmi": "^1.5.3",
    "@tanstack/vue-query": "^5.62.2",
    "@wagmi/vue": "^0.0.68",
    ...
    "nuxt": "^3.14.1592",
    "vue": "^3.5.13",
```

### Node.js Version

v22.6.0

### Package Manager

yarn@4.5.3

**Comment by arein** [comment]
@creazy231 I'm pretty sure this is part of the fix we're releasing in 1.5.4 today - can you check tomorrow if it's fixed on 1.5.4?

**Comment by naincrypto** [comment]
how to rectify this i am not under stand what to do
